source/app/service-providers/commands/file-new.ts

```diff
@@ -146,17 +165,15 @@ export default class FileNew extends ZettlrCommand {
           await this._app.fsal.removeFile(absPath)
         }
       }
-
       // First create the file
       await this._app.fsal.writeTextFile(absPath, '')
-
-      // And directly thereafter, open the file
```

Comment:
Please keep this comment. (In general, please do not delete comments unless they become invalid due to any change)